index.html

```diff
@@ -502,6 +502,11 @@ <h3>Character</h3>
 &lt;/metadata&gt;
 ...
         </pre>
+            <li>All <code>ttm:agent</code> elements SHOULD be contained in a single <code>metadata</code> element.
+              <div class="note">There can be multiple <code>metadata</code> elements in the <code>head</code> element,
```

Comment:
Element convention regardless of the path spec suggestion:

```suggestion
            <li>All <code>&lt;ttm:agent&gt;</code> elements SHOULD be contained in a single <code>&lt;metadata&gt;</code> element.
              <div class="note">There can be multiple <code>&lt;metadata&gt;</code> elements in the <code>&lt;head&gt;</code> element,
```